managed/CounterStrikeSharp.API/NativeEntity.cs

```diff
@@ -0,0 +1,21 @@
+using System.Runtime.InteropServices;
+using CounterStrikeSharp.API.Modules.Entities;
+using CounterStrikeSharp.API.Modules.Utils;
+
+namespace CounterStrikeSharp.API;
+
+public abstract class NativeEntity : NativeObject
+{
+    public new IntPtr Handle => EntitySystem.GetEntityByHandle(EntityHandle) ?? 0;
```

Comment:
what about using `IntPtr.Zero` instead of 0? its the same just it got my attention

Reply:
I just got lazy haha

Reply:
Fixed it now.